content/docs/command-reference/import-url.md

```diff
@@ -51,7 +51,7 @@ corresponding local path in the <abbr>workspace</abbr>. It records enough
 metadata about the imported data to enable DVC efficiently determining whether
 the local copy is out of date.
 
-`dvc repro` doesn't check and/or update generated `.dvc` files, use `dvc update`
+`dvc repro` doesn't check and/or update generated `.dvc` files. Use `dvc update`
```

Comment:
Just thought that the two would be more readable here given that their subjects are different.

Reply:
No, these are separate sentences. Also, out of scope.

Reply:
"and/or" should just be "or" though, please change that since we're looking at this already.